src/Features/CSharp/Portable/UseDefaultLiteral/CSharpUseDefaultLiteralCodeFixProvider.cs

```diff
@@ -52,6 +53,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
 
             var parseOptions = (CSharpParseOptions)document.Project.ParseOptions;
             var options = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
+            var preferSimpleDefaultExpression = options.GetOption(CSharpCodeStyleOptions.PreferSimpleDefaultExpression).Value;
```

Comment:
Is the .Value earlier now safe? Previously it wouldn't have ran unless something else happened earlier?